src/utils/db/car_util.rs

```diff
@@ -1,12 +1,27 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+use cid::Cid;
 use futures::{Stream, StreamExt, TryStreamExt};
+use fvm_ipld_blockstore::Blockstore;
 use tokio::io::{AsyncBufRead, AsyncSeek};
 
 use crate::ipld::CidHashSet;
 use crate::utils::db::car_stream::{CarBlock, CarStream};
 
+pub async fn load_car<R, B>(db: &B, reader: R) -> Result<Vec<Cid>, anyhow::Error>
+where
+    B: Blockstore,
+    R: tokio::io::AsyncSeek + tokio::io::AsyncBufRead + Send + Unpin,
+{
+    let mut stream = CarStream::new(tokio::io::BufReader::new(reader)).await?;
```

Comment:
Do we want to `impl Sink<CarBlock> for T where T: Blockstore`?

Reply:
I'm not sure about that.